src/main/java/nl/tudelft/scrumbledore/Constants.java

```diff
@@ -22,7 +22,7 @@
 public final class Constants {
 
   /**
-   * Logger constructor, which should be used only once since this is a Singleton class.
+   * Constants constructor, which should be used only once since this is a Singleton class.
```

Comment:
This is a static utility class, therefore the constructor is never used. <review